routes/api.php

```diff
@@ -13,6 +13,4 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
```

Comment:
Reálne sa auth nikde nepoužíva, bol to len pozostatok vygenerovanej kostry Laravel projektu.